src/app/main/component/shared/components/calendar-base/calendar-base.component.ts

```diff
@@ -317,13 +320,20 @@ export class CalendarBaseComponent implements OnDestroy {
     const date = this.formatDate(isMonthCalendar, dayItem);
     const habits = this.getHabitsForDay(this.userHabitsList, date);
     const pos = event.target.getBoundingClientRect();
+    this.breakpointObserver.observe(['(max-width: 912px)']).subscribe((result: BreakpointState) => {
+      if (result.matches) {
+        horisontalPositioning = '';
+      } else {
+        horisontalPositioning = pos.left + '300px';
+      }
+    });
```

Comment:
use ternary operator here:
horisontalPositioning = result.matches ? '' : pos.left + '300px'

Reply:
chaged it